cli/fathom_web/commands/train.py

```diff
@@ -54,12 +54,34 @@ def learn(learning_rate, iterations, x, y, confidence_threshold, num_prunes, val
     if stopped_early:
         print(f'Stopping early at iteration {t}, just before validation error rose.')
 
+    print(f'Successfully trained model using threshold {confidence_threshold}.')
+    optimal_thresholds, max_accuracy = find_optimal_threshold(y, y_pred, num_prunes, threshold_incr=0.05)
+
+    print(f'Suggested threshold(s) are: {optimal_thresholds}.')
+    print(f'All listed thresholds have the same accuracy value of {max_accuracy}')
```

Comment:
We won't need to say this, since the giant accuracy chart already displays accuracy.

Reply:
That chart will display the accuracy for the threshold at 0.5?

Reply:
Well, `print_per_tag_report` will use whatever confidence threshold you pass in, and we should pass in our optimized threshold. Similar, `pretty_accuracy` (which prints the table of numbers with the confusion matrix and so on) [takes the output of `accuracy_per_tag`](https://github.com/mozilla/fathom/blob/489e78f2fd857ad64b259933b0566b7e4373f3c3/cli/fathom_web/commands/train.py#L269-L275), which itself takes the threshold. So, if we pass our optimized threshold to that as well, it should be reflected in all our output.